docker/opensource.md

```diff
@@ -86,6 +86,15 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # composer install
 # phpunit
 ```
-
+## Plagiarism-checker-Python
+$ git clone https://github.com/Kalebu/Plagiarism-checker-Python
+$ cd Plagiarism-checker-Python
+$ docker run -it --name Renormalizer-devPlagiarism-checker-Python -w /opt -v %cd%:/opt python:latest bash
+$ pip3 install -r requirements.txt
+# running the project
+$ Plagiarism-checker-Python-> python3 app.py
```

Comment:
What does this `Plagiarism-checker-Python-> ` prefix do here?
Did you manage to run the code this way?

Reply:
This is the name of the Directory
I just wanted that the prompt will have the same name as the Directory

Reply:
Inside the docker container that is not the name of the directory and besides `$` was already there to indicated the prompt

Reply:
it is the name of the directory and the name of the container
![Screenshot 2023-05-04 114536](https://user-images.githubusercontent.com/51055096/236155313-60d29749-420a-4ce5-aa37-317fac04ed1c.png)

Reply:
That's confusing here.